proto/telemetry.proto

```diff
@@ -7,8 +7,7 @@ option go_package = "risingwavelabs.com/risingwave/proto/telemetry";
 enum MetaBackend {
   META_BACKEND_UNSPECIFIED = 0;
   META_BACKEND_MEMORY = 1;
-  reserved 2;
-  reserved "META_BACKEND_ETCD";
```

Comment:
telemetry service still receive requests from legacy clusters and it is still possible getting reports from a ETCD backend cluster. 
I believe the field is changed by mistake in https://github.com/risingwavelabs/risingwave/pull/18621 
There is no breaking change in the proto file because there is no updates on the server side since then.